### PR TITLE
Palo Alto  FW/PAN - service object , service group object commands

### PR DESCRIPTION
**Pull request by bakatzir**
and deletion of non existing objects error handling (code 17 from palo alto)

PanoramaCommitConfiguration fixes
modules.name - change to module.brand
filter jobid to not run GenericPooling if not necessary(did the same in atd -detonate file playbook)

## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/14462, https://github.com/demisto/etc/issues/15115

## Does it break backward compatibility?
   - No

## Must have
- [x] Tests 
- [x] Documentation - https://github.com/demisto/etc/issues/15223
- [ ] Code Review

### CODE REVIEW

File: Integrations/integration-Panorama.yml [hunk 1]
```diff
@@ -961,6 +966,443 @@ script:
...
+
+            pretty_services_arr.append(pretty_service)
+
+        return pretty_services_arr
+
+
+    @logger
+    def panorama_list_services():
+        params = {
+            'action': 'get',
```
Comment (yuvalbenshalom) [inline Integrations/integration-Panorama.yml]: y u no getting params in command function?

File: Integrations/integration-Panorama.yml [hunk 2]
```diff
@@ -147,11 +148,15 @@ script:
             return_error('Request Failed. with status: ' + str(result.status_code) + '. Reason is: ' + str(result.reason))
 
         jresult = json.loads(xml2json(result.text))
+
+        #panorama_check_response(jresult)
```
Comment (yuvalbenshalom) [inline Integrations/integration-Panorama.yml]: what is this for?

File: Integrations/integration-Panorama.yml [hunk 3]
```diff
@@ -961,6 +966,443 @@ script:
...
+            pretty_service['DestinationPort'] = service['protocol'][protocol]['port']
+        if 'source-port' in service['protocol'][protocol]:
+            pretty_service['SourcePort'] = service['protocol'][protocol]['source-port']
+
+        return pretty_service
+
+
+    @logger
+    def panorama_get_service(service_name):
+        params = {
```
Comment (yuvalbenshalom) [inline Integrations/integration-Panorama.yml]: y u no getting params in command function?

File: Integrations/integration-Panorama.yml [hunk 4]
```diff
@@ -961,6 +966,443 @@ script:
...
+                'Name': service_group['@name'],
+                'Services': service_group['members']['member']
+            }
+            pretty_service_groups_arr.append(pretty_service_group)
+
+        return pretty_service_groups_arr
+
+
+    @logger
+    def panorama_list_service_groups():
```
Comment (yuvalbenshalom) [inline Integrations/integration-Panorama.yml]: y u no getting params in command function?

File: Integrations/integration-Panorama.yml [hunk 5]
```diff
@@ -961,6 +966,443 @@ script:
...
+    def prettify_service_group(service_group):
+        pretty_service_group = {
+            'Name': service_group['@name'],
+            'Services': service_group['members']['member']
+        }
+        return pretty_service_group
+
+
+    @logger
+    def panorama_get_service_group(service_group_name):
```
Comment (yuvalbenshalom) [inline Integrations/integration-Panorama.yml]: same

File: Integrations/integration-Panorama.yml [hunk 6]
```diff
@@ -961,6 +966,443 @@ script:
...
+            'HumanReadable': 'Service was added successfully',
+            'EntryContext': {
+                  "Panorama.Services(val.Name == obj.Name)": service_output
+            }
+        })
+
+
+    @logger
+    def panorama_delete_service(service_name):
+        params = {
```
Comment (yuvalbenshalom) [inline Integrations/integration-Panorama.yml]: same

**Comment by bakatzir** [comment]
@yuvalbenshalom regarding  getting params in command function
I divided it that the demisto commands, outputs and hr are in: _command, the request logic is in the function without that prefix. why move it?

File: Integrations/integration-Panorama.yml (hunk 2 above)
Comment (bakatzir) [reply Integrations/integration-Panorama.yml]: forgot to delete it. fixed.

**Comment by yuvalbenshalom** [comment]
@bakatzir 
the convention is that arguments are fetched inside the `_command` function and are being passed as arguments to the business logic function.

**Pull request merged**